src/handlers/ens.ts

```diff
@@ -220,7 +220,7 @@ export const fetchSuggestions = async (
       avatar = await fetchENSAvatar(ens, {
         cacheFirst: true,
       });
-      prefetchENSAddress(ens, { cacheFirst: true });
+      prefetchENSAddress({ name: ens }, { staleTime: Infinity });
```

Comment:
not following the need of having `staleTime` here, from the [doc](https://www.notion.so/rainbowdotme/Async-State-RFC-711800896f9a458889fc4ffad4de271a#beab96145f85425782627ebe11680069) staleTime should be ignored since there's no `cacheTime` defined inside `prefetchENSAddress`

also if the staleTime is infinity we wouldn't do any background refresh which is probably something we don't want since ens names can change

am i missing something?

Reply:
Yeah, `Infinity` was just to make it behave the same as `cacheFirst`. But maybe we should only make it stale for like 10 minutes or something.